src/db/migrations/1691398917868-AddAdminToMinter.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddAdminToMinter1691398917868 implements MigrationInterface {
+  name = 'AddAdminToMinter1691398917868';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE \`minters\` ADD \`adminAddress\` varchar(62) NOT NULL`);
```

Comment:
In the future / on different environments, we might have addresses with a longer HRP (instead of `erd1`).

Reply:
will address then :D